src/views/ProductInventoryDetail.vue

```diff
@@ -0,0 +1,779 @@
+  checkmarkOutline,
+  ellipsisVerticalOutline,
+  globeOutline,
+  listOutline,
+  locationOutline,
+  shareOutline,
+  shirtOutline,
+  sendOutline,
+  storefrontOutline,
+  syncOutline,
+  ticketOutline,
+} from 'ionicons/icons';
+import Image from '@/components/Image.vue';
+import EditQuantityModal from '@/components/EditQuantityModal.vue';
+import LocationPopover from '@/components/LocationPopover.vue';
+import PurchaseOrderPopover from '@/components/PurchaseOrderPopover.vue';
+import FulfillmentSettingsPopover from '@/components/FulfillmentSettingsPopover.vue';
+
+export default defineComponent({
+  name: 'ProductInventoryDetail',
+  components: {
+    IonBackButton,
+    IonButton,
+    IonButtons,
+    IonCard,
+    IonCardHeader,
+    IonCardTitle,
+    IonCheckbox,
+    IonChip,
+    IonContent,
+    IonHeader,
+    IonIcon,
+    IonItem,
+    IonLabel,
+    IonList,
+    IonListHeader,
+    IonNote,
+    IonPage,
+    IonSearchbar,
+    IonSegment,
+    IonSegmentButton,
+    IonTitle,
+    IonThumbnail,
+    IonToggle,
+    IonToolbar,
+    Image
+  },
+  methods: {
+    segmentChanged(ev: CustomEvent) {
+      this.segment = ev.detail.value;
+    },
+    async editQuantity() {
+      const editmodal = await modalController.create({
+        component: EditQuantityModal
+      });
+      return editmodal.present();
+    },
+    async openLocationPopover(ev: Event) {
+      const popover = await popoverController.create({
+        component: LocationPopover,
+        event: ev,
+        translucent: true,
+        showBackdrop: false,
+      });
+      return popover.present();
+    },
+    async openPopover(ev: Event) {
+      const popover = await popoverController.create({
+        component: PurchaseOrderPopover,
+        event: ev,
+        translucent: true,
+        showBackdrop: false,
+      });
+      return popover.present();
+    },
+    async openFulfillmentSettings(ev: Event) {
+      const popover = await popoverController.create({
+        component: FulfillmentSettingsPopover,
+        event: ev,
+        translucent: true,
+        showBackdrop: false,
+      });
+      return popover.present();
+    },
+  },
+  setup() {
+    const router = useRouter();
+    const segment = ref("locations");
+
+    return {
+      businessOutline,
+      calendarOutline,
+      checkmarkOutline,
+      ellipsisVerticalOutline,
+      globeOutline,
+      listOutline,
+      locationOutline,
+      shareOutline,
+      shirtOutline,
+      sendOutline,
+      storefrontOutline,
+      syncOutline,
+      ticketOutline,
+      router,
+      segment,
+    }
+  }
+});
+</script>
+
+<style scoped>
+.product {
+  display: grid;
+  grid-template-columns: 254px auto;
+  justify-items: start;
+  align-items: center;
+}
+
+.product-info,
+.variant-info {
+  width: 343px;
+}
+
+.variant,
+.actions {
+  display: grid;
+  grid-template-columns: repeat(2, auto);
+  justify-content: space-between;
+  align-items: center;
+}
+
+.text-center {
```

Comment:
is there a better alternative here. like some kind of selector?

Reply:
I have used selector here sir for aligning text of all ion-label in `open-orders` class but if I do so it is aligning text of first row labels also which we dont want that is why I have explicitly added it to desired labels.

Reply:
what about a not first child selector?

Reply:
or maybe just add another selector that selects first child and resets text align property

Reply:
Yes I am exploring it sir

Reply:
Used the not first child selector.

![localhost_8100_product-inventory-detail](https://user-images.githubusercontent.com/82817616/147821512-60d0db7f-df9c-4e20-bac2-b84391cbeb0d.png)